### ISSUE TITLE
Not able to keyboard navigate into subtab items

### ISSUE THREAD

**Issue by LianaHarris360** [learningequality/kolibri#10426]
## Observed behavior
Reported by: @jtamiace in the alpha-12 bug bash

Cannot navigate into subtab items on menu and tabbing goes outside of the menu to unseen elements.

https://user-images.githubusercontent.com/46411498/230913262-f9f32824-a3dd-4398-ba63-014d842fd1c4.mp4

## Expected behavior
Pressing tab on keyboard navigates to subtab menu items.

## User-facing consequences
Users that rely on keyboard navigation are unable to access some pages.

## Context
0.16.0-alpha12

**Comment by LianaHarris360** [comment]
Pressing `Option + Tab` or updating the Keyboard navigation settings fixes this issue.
